transforms/builder/addAttribute.js

```diff
@@ -0,0 +1,57 @@
+const addAttribute = (path, attribute, value) => (
+  fileInfo,
+  api,
+  options,
+  j,
+  root
+) => {
+  const components = root.findJSXElementsByImport(path);
+
+  if (components.size() === 0) {
+    // If the component is not imported, skip this file
+    return;
+  }
+  if (!components) {
+    return false;
+  }
+
+  let didUpdate = false;
+  components.forEach((component) => {
+    if (
+      !j(component.value.openingElement.attributes)
+        .find(j.JSXIdentifier, { name: attribute })
+        .size()
+    ) {
+      if (typeof value === "string") {
+        component.value.openingElement.attributes.push(
+          j.jsxAttribute(j.jsxIdentifier(attribute), j.literal(value))
+        );
+        didUpdate = true;
+      }
+
+      if (
+        (typeof value !== "string" && typeof value !== "boolean") ||
+        (typeof value === "boolean" && value === false)
+      ) {
+        component.value.openingElement.attributes.push(
+          j.jsxAttribute(
+            j.jsxIdentifier(attribute),
+            j.jsxExpressionContainer(j.literal(value))
+          )
+        );
+        didUpdate = true;
```

Comment:
Same as above, could just be `return true` so we don't run over the next if statement